src/lib/stylejson.ts

```diff
@@ -1,9 +1,8 @@
 /* eslint-disable @typescript-eslint/ban-types */
 import { URL } from 'url'
-import {
-  validate as validateStyleJSON,
-  StyleSpecification as StyleJSON,
-} from '@maplibre/maplibre-gl-style-spec'
+// @ts-ignore
+import { validate as validateStyleJSON } from '@maplibre/maplibre-gl-style-spec'
+import { StyleSpecification as StyleJSON } from '@maplibre/maplibre-gl-style-spec/types.g'
```

Comment:
Is this a typo? `/types.g`?

Reply:
not a typo 😅 although i'm finding that this is causing some issues with the `npm run dev` command (works okay when removing the `--transpile-only` flag, so may just vendor this file anyways